src/datamapper/AusenciaJpaController.java

```diff
@@ -44,17 +47,17 @@ public void create(Ausencia ausencia) {
         }
     }
 
-    public void edit(Ausencia ausencia) throws NonexistentEntityException, Exception {
+    public void edit(Ausencia ausencias) throws NonexistentEntityException, Exception {

```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Line is longer than 80 characters (found 87).](https://www.codacy.com/app/calebepb/substituicao/file/2738539596/issues/source?bid=3308597&fileBranchId=3353540#l50)